pages/suggestions-page.js

```diff
@@ -0,0 +1,10 @@
+import SuggestedColor from "@/components/SuggestedColor";
+
+export default function SuggestionPage({ color }) {
+  return (
+    <>
+      <SuggestedColor text="Color Suggestion Nr. 1" color={color.color1} />
```

Comment:
you could work here with {children} instead of your text-prop
```suggestion
      <SuggestedColor color={color.color1} >Color Suggestion Nr. 1</SuggestedColor>
```

Reply:
fixed 👍